counting_elements.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Uniter PHP Counting Current Elements Demo</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="/assets/jquery-full.js"></script>
```

Comment:
jQuery should be embedded in the compiled `bundle.js`, no need to require it separately here please